server/src/main/java/org/candlepin/audit/MessageAddress.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2009 - 2018 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.audit;
+
+/**
+ * Defines the addresses that are used when sending and receiving messages from Artemis.
+ */
+public final class MessageAddress {
```

Comment:
Would this be better defined as an enum?

Reply:
I considered using an enum here, but I figured I'd have to add the supporting field to get the address value and it didn't really require a type. Figured this was a simple approach to store/define String constants that required no specific type. I'm Ok with moving it to an enum if you feel strongly about it.

I suppose I could move each to the MessageHandler that requires it and reference it from there as well.